README.md

```diff
@@ -277,7 +277,10 @@ new initializer in `config/initializers/resque.rb` with the following content:
 ```ruby
 # config/initializers/resque.rb
 require 'airbrake/resque/failure'
-Resque::Failure.backend = Resque::Failure::Airbrake
+require 'resque/failure/redis'
+require 'resque/failure/multiple'
+Resque::Failure::Multiple.classes = [Resque::Failure::Redis, Resque::Failure::Airbrake]
+Resque::Failure.backend = Resque::Failure::Multiple
```

Comment:
I feel like this information shouldn't be here. We don't want to enforce Redis and we only want to inform how to configure Airbrake, so no need to mention third parties. Resque has integration instructions with multiple backends, so we're good.